docs/huggingface_models.md

```diff
@@ -0,0 +1,29 @@
+# Using HuggingFace Models
+
+HELM can be used to evaluate models on [Hugging Face Model Hub](https://huggingface.co/models).
+
+To use models from Hugging Face Model Hub, add the Hugging Face model IDs to the `--register-huggingface-models` flags to `helm-run`. This will make the corresponding Hugging Face models available to use in your run spec descriptions. In the run spec description, use the Hugging Face model ID as the model name.
+
+To use revision of a model other than the default main revision, append a `#` followed by the revision name to the model ID passed to the `--register-huggingface-models` flag. Note that in the run spec description, the model name should not include `#` or the revision name.
+
+Restrictions: `--register-huggingface-models` currently does not support models without a namespace (e.g. `bert-base-uncased`).
```

Comment:
Maybe give an autoregressive LM as an example.  In general, should clarify we can't run any HF model, but only HF models that are LMs - how can we tell?

Reply:
yeah it's probably worth documenting that we're specifically going to use AutoModelForCausalLM to load it

Reply:
Added note that only `AutoModelForCausalLM` is supported.